tests/FixerDefinition/CodeSampleTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\FixerDefinition;
+
+use PhpCsFixer\FixerDefinition\CodeSample;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+final class CodeSampleTest extends \PHPUnit_Framework_TestCase
+{
+    public function testImplementsCodeSampleInterface()
+    {
+        $reflection = new \ReflectionClass('PhpCsFixer\FixerDefinition\CodeSample');
```

Comment:
let me repeat the question
what is a benefit for checking that class implements an interface?
if that would be violated, the compiler-time parameter typechecking will raise an error and test would not pass as well.